test/data/manifests/fedora-ostree-oci-bootiso.mpp.yaml

```diff
@@ -0,0 +1,428 @@
+version: '2'
+pipelines:
+  - mpp-import-pipelines:
+      path: fedora-vars.ipp.yaml
+  - mpp-import-pipeline:
+      path: fedora-build-v2.ipp.yaml
+      id: build
+    runner:
+      mpp-format-string: org.osbuild.fedora{release}
+  - name: anaconda-tree
+    build: name:build
+    stages:
+      - type: org.osbuild.rpm
+        inputs:
+          packages:
+            type: org.osbuild.files
+            origin: org.osbuild.source
+            mpp-depsolve:
+              architecture: $arch
+              module-platform-id: $module_platform_id
+              repos:
+                mpp-eval: repos
+              packages:
+                - anaconda
+                - anaconda-widgets
+                - kexec-tools-anaconda-addon
+                - anaconda-install-env-deps
+                - dnf
```

Comment:
You should be able to pick up a bunch of these by using `anaconda-install-img-deps` which pulls in things like the xorg server, gnome-kiosk, etc.